vars/setupAndRunScaleTest.groovy

```diff
@@ -33,10 +33,10 @@ def call(){
                 oc get pods -o wide -n openshift-image-registry ;
                 sleep 900 ;
                 oc get pods -o wide -n openshift-image-registry ;
-                echo './clusterbuster -d ${SCALE_NUM_OF_DEPLOYMENTS} -B ltccci ;while [ `oc get pods -n ltccci-0 | grep -v NAME | grep -v Running | wc -l` -ne 0 ];do sleep .0000000001;done' > deployments.sh;
+                echo './clusterbuster -d ${SCALE_NUM_OF_DEPLOYMENTS} -B ocp-cicd ;while [ `oc get pods -n ocp-cicd-0 | grep -v NAME | grep -v Running | wc -l` -ne 0 ];do sleep .0000000001;done' > deployments.sh;
```

Comment:
this change is not necessary , its just the name of the pods

Reply:
@sudeeshjohn Addressed review comments